### PR TITLE
Add jobs API methods

### PR DESCRIPTION
**Pull request by jmcphers**
This change adds rstudioapi wrappers for RStudio's Jobs Pane.

### CODE REVIEW

File: R/jobs.R [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+
+#' Run R Script As Job
+#'
+#' Starts an R script as a background job.
+#' 
+#' @param path The path to the R script to be run
+#' @param encoding The text encoding of the script, if known.
+#' @param workingDir The working directory in which to run the job.
```
Comment (kevinushey) [inline R/jobs.R]: What working directory is used by default (when the parameter is `NULL`)?

File: R/jobs.R [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+#'    have an informational link in the Jobs pane rather than an output
+#'    display, and clicking the link will invoke the \code{info} action.}
+#' }
+#'
+#' @export
+jobAdd <- function(name, status = "", progressUnits = 0L, actions = NULL, running = FALSE, 
+                   autoRemove = TRUE, show = TRUE) {
+    callFun("addJob", name          = name,
+                      status        = status,
+                      progressUnits = progressUnits,
```
[the remark this answers is not in the document]
Comment (jmcphers) [reply R/jobs.R]: We should. I'll make that change in the IDE.

File: R/jobs.R [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                      running       = running,
+                      autoRemove    = autoRemove,
+                      show          = show)
+}
+
+#' Remove a Job
+#'
+#' Remove a job from RStudio's Jobs pane.
+#' 
+#' @param job The ID of the job to remove
```
[the remark this answers is not in the document]
Comment (jmcphers) [reply R/jobs.R]: Added.

File: R/jobs.R [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+jobAddOutput <- function(job, output, error = FALSE) {
+    callFun("addJobOutput", job = job, output = output, error = error)
+}
+
+
+#' Run R Script As Job
+#'
+#' Starts an R script as a background job.
+#' 
+#' @param path The path to the R script to be run
```
[the remark this answers is not in the document]
Comment (jmcphers) [reply R/jobs.R]: Added.

File: R/jobs.R (hunk 1 above)
Comment (jmcphers) [reply R/jobs.R]: A good question! It's the parent dir of the script. I'll update the documentation so this is clear.

**Pull request merged**